README.md

```diff
@@ -34,7 +34,7 @@ curl https://raw.githubusercontent.com/puneetsl/lotion/master/setup.sh > setup.s
 chmod +x setup.sh
 
 # Run (with sudo for global installation, without sudo for local installation)
-[sudo] ./setup.sh native
+[sudo] ./setup.sh nativej

```

Comment:
Was this intentionally changed to nativej or just a typo?

Reply:
whoops! not sure it ended up there

Reply:
Good catch! I'll fix this